sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential_async.py

```diff
@@ -22,8 +23,9 @@ class CommunicationTokenCredential(object):
     ON_DEMAND_REFRESHING_INTERVAL_MINUTES = 2
 
     def __init__(self,
-            communication_token_refresh_options
+            token # type: str
```

Comment:
`CommunicationTokenRefreshOptions` has two arguments: `tokenRefresher` and `token`. Let's make sure we pass them both here as well

Reply:
You can also remove the `CommunicationTokenRefreshOptions` type altogether and move `_token` and `_token_refresher` to `CommunicationTokenCredential`

Reply:
@RezaJooyandeh Haven't removed the CommunicationTokenRefreshOptions type completely. Added the 2 fields in CommunicationTokenCredential